src/data/components.json

```diff
@@ -1057,11 +1057,11 @@
         "notAvailable": false
       }
     },
-    "Content group horizontal": {
-      "url": "./content-group-horizontal",
+    "Content block horizontal": {
+      "url": "./content-block-horizontal",
       "designLink": "https://ibm.box.com/s/rv5kmm0a4d9ded74v6yfk8ad5id5ghmx",
       "functionalLink": "https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Content-block-horizontal",
```

Comment:
Now that the naming is aligned again I think we can get rid of this functional link. The component builds the link based on its name.

```suggestion
```